examples/test_examples.py

```diff
@@ -20,6 +20,7 @@ def wait_true(wait_time, func, *args, **kwargs):
 
     return False
 
+
```

Comment:
\<sarcasm\> How *dare* you bring empty lines into this PR. Don't you care about the diff drift!? \</sarcasm\>